src/filters/util.js

```diff
@@ -3,26 +3,18 @@
 angular.module('openshiftCommonUI')
   .filter("toArray", function() {
     return function (items) {
-      if (!items) {
-        return [];
-      }
-
-      if (angular.isArray(items)) {
-        return items;
-      }
-
-      var itemsArray = [];
-      angular.forEach(items, function (item) {
-        itemsArray.push(item);
-      });
-
-      return itemsArray;
+      return _.toArray(items);
+    };
+  })
+  .filter('size', function() {
+    return function(items) {
+      return _.size(items);
     };
   })
-  .filter('hashSize', function() {
-    return function(hash) {
-      if(!hash) { return 0; }
-      return Object.keys(hash).length;
+  .filter('hashSize', function($log) {
+    return function(items) {
+      $log.warn('hashSize filter is now deprecated. Use size filter instead.');
+      return _.size(items);
```

Comment:
OK, I will put the original implementation back and remove the warning. Folks will just have to know to use size rather than hashSize and to update along the way.